### DISCUSSION TITLE
Diagnosing movement pauses

### DISCUSSION THREAD

**Discussion by v-mwalk** [input-leap/input-leap#2074] [Q&A]
I have two Macs, using Input Leap (🙇 ).  Mostly it works well, however, there i a relatively frequent 'pause' - mouse stops for upto 3-4 seconds.

There could be a whole bunch of Leap and non-Leap related reasons for this.  Mac process (although have 'renice'd this to 20), network etc...

My question is - how to debug this?  I have tried the logs debug1/debug2 (or some reason debug3 doesnt like me) and, while not sure what sort of log message I'm looking for - server saying 'tried to sent a movement packet but failed/delayed'? - have not been able to find a way.

Anyone with ideas? (I am relatively techy 'low level' - ie nearer the metal - btw)

**Comment by SergeyGadzhilov** [comment]
Try to turn off universal clipboard.
